// Create the CustomError class here

// Instantiate CustomError and throw it
// const errorInstance = new CustomError();
// throw errorInstance;
